CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(cityflow)
 set(CMAKE_CXX_STANDARD "11" CACHE STRING "")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -DRAPIDJSON_HAS_STDSTRING=1")
 set(CMAKE_CXX_FLAGS_RELEASE "-O2")
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_DEBUG "-g -D_GLIBCXX_DEBUG")
```

Comment:
Can you explain why you need this flag?

Reply:
It is unneccesary after I installed libstdc++-dbg. Thanks to your reminder,